argcomplete/__init__.py

```diff
@@ -8,6 +8,7 @@
 from .compat import USING_PYTHON2, str, sys_encoding, ensure_str, ensure_bytes
 from .completers import FilesCompleter, SuppressCompleter
 from .my_argparse import IntrospectiveArgumentParser, action_is_satisfied, action_is_open, action_is_greedy
+from .shellintegration import shellcode # noqa
```

Comment:
I have added noqa to avoid a linter error that is not actually an error. See https://bugs.launchpad.net/pyflakes/+bug/1178905